doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+       
+       .. altair-plot::
+         :hide-code:
+
+         import altair as alt
+         from vega_datasets import data
+
+         source = data.stocks()
+         col_range = [
+             "#E69F00",
+             "#57B4E9",
+             "#019E73",
+             "#F0E442",
+             "#0072B2",
+         ]
+         col_sort = [
+             "GOOG",
+             "AAPL",
+             "IBM",
+             "AMZN",
+             "MSFT",
+         ]
+
+         alt.Chart(
+             source, width=200, height=200
+         ).mark_bar().encode(
+             x=alt.X('symbol:N', sort='-y'),
+             y=alt.Y('price:Q').aggregate('sum'),
+             color=alt.Color(
+                 "symbol:N", sort=col_sort
+             ).scale(range=col_range),
+         )
+
+     - When unique categories are not distinguishable by their color usage, 
+       it can cause confusion and misunderstandings.
+
+       .. altair-plot::
+         :hide-code:
+
+         source = data.stocks()
+
+         alt.Chart(
+             source, width=200, height=200
+         ).mark_bar().encode(
+             x=alt.X('symbol:N', sort=col_sort),
+             y=alt.Y('price:Q').aggregate('sum'),
+             color=alt.Color("symbol:N").scale(
+                 scheme="purples"
+             ),
+         )
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Explain colors using a colorbar
+     - No understanding of used colors
+
+   * - When you use colors in your visualization, it's important to explain
+       what each color means so that people can understand your message better.
+  
+       .. altair-plot::
+         :hide-code:
+
+         import altair as alt
+         from vega_datasets import data
+         
+         source = data.iowa_electricity()
+         
+         alt.Chart(
+            source, width=200, height=200
+         ).mark_area().encode(
+             x="year:T",
+             y="net_generation:Q",
+             color=alt.Color("source:N").legend(
+                orient="bottom",
+                direction="horizontal",
+                columns=2
+             )
+         )
+
+     - If you do not explain what your colors mean, people might be confused
+       and not understand your visualization properly. 
+       
+       .. altair-plot::
+         :hide-code:
+
+         import altair as alt
+         from vega_datasets import data
+         
+         source = data.iowa_electricity()
+         
+         alt.Chart(
+            source, width=200, height=200
+         ).mark_area().encode(
+             x="year:T",
+             y="net_generation:Q",
+             color=alt.Color("source:N").legend(
+                None
+             )
+         )
+
+
+Color schemes
+~~~~~~~~~~~~~
+
+This sections presents the three primary color schemes used in data
+visualization: sequential, diverging, and categorical. 
+
+Sequential schemes
+^^^^^^^^^^^^^^^^^^
```

Comment:
I think it is great that we are showing how each palette look for common color vision deficiencies here! I think it would also be great to include which ones are perceptually uniform, but maybe that is not possible? Or a small line chart / spark line maybe could work for the luminiosty? I also think that is nice that we show an example of the discrete version of the scheme, but maybe we could more clearly label that this is what it would look like for ordinal data (which I think is the main scenario where they would be used?)?

Reply:
It also seems like the sequential multi-hue schemes (e.g. viridis) are missing?